src/TestSuite/IntegrationTestCase.php

```diff
@@ -211,10 +211,17 @@ public function session(array $data)
      *
      * @param string $name The cookie name to use.
      * @param mixed $value The value of the cookie.
+     * @param string|bool|null $encryption By default, the CookieComponent
+     *   isn't be used. If specified the encryption mode like 'aes' or false,
+     *   the CookieComponent is used to encrypt and json_encode if array.
```

Comment:
CookieComponent::write() can accept an array which will encode to json.
I think some beginners try to do `$this->cookie('anyKey', ['key' => 'value']);`.
But, it doesn't work because `cookie()` method does not json_encode by default, unlike CookieComponent. 
The default works of `cookie()` should not be changed, so I think it is better way to introduce `false` as same as the `encryption` of `CookieComponent`.